### PR TITLE
Bugfix/dapp transfer eth screen mobile chaos

### LINKED ISSUE #2421: Account screen looks a bit out of order in LC mobile
## Steps to Reproduce
Please refer to link here to get started with using LC on mobile 
https://github.com/raiden-network/light-client/issues/2400#issuecomment-734294170

## Expected Result
- The background thoughout the Account screen has to be uniform 

## Actual Result
- raiden_lc_account_screen

### LINKED ISSUE #2422: Editing ether value in the Transfer Eth screen gets white out in LC mobile
## Steps to Reproduce
https://github.com/raiden-network/light-client/issues/2400#issuecomment-734294170 

## Expected Result
- The ether amount being entered by the user should be visible to him 

## Actual Result
- Transfer eth screen the amount is not visible when editing it. Its gets white out due to the background.
transfer_eth_screen

### PR DESCRIPTION
**Pull request by weilbith**
**Thank you for submitting this pull request :)**

Fixes #2422
Fixes #2421 (added later to make use of closing issue automatically)

**Short description**
The current usage of the account route as second route view is madness. It is absolute crazy. But as fixing this is a lot of work, this is just a workaround for the issues. The root cause is that the viewport size changes, when a mobile virtual keyboard gets shown. Thereby the content does not fit anymore and a the result the layout is messed and more.

I did some additional minor design changes to make it look not so compressed on mobile.

This also fixes #2421 and eventually makes #2563 obsolete. The downside is, that my scrolling applies to the whole application route container, including the header. This eventually looks ugly. This might be fixable, but just drives us even further into the madness of how the routes are organized. So I did not add it. @taleldayekh please decide on this. I can't think objectively here. If we take this over your PR, please remind me to extend the CHANGELOG for this issue as well.

**Definition of Done**

- [x] Steps to manually test the change have been documented
- [x] Acceptance criteria are met
- [x] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1. Open dApp (e.g. with PR serve tool)
2. Switch to mobile view and select a mobile phone by Google (e.g. Nexus 5X, maybe needs to be added to the list) as those support tests with virtual keyboards
2. Connect and open the account route
3. Make sure you can scroll down the menu entries without a broken background (issue #2421)
4. Select the `Raiden Account`entry/button
5. On the mobile view header bar of the dev tools, click on the rotating phone and select `Portrait - Keyboard`
6. Make sure everything looks fine and you can enter an ETH amount

### CODE REVIEW

**Comment by agatsoh** [comment]
I can confirm this works on a real mobile. I just did a test using ngrok and tested this PR from local using ngrok.

lc_ngrok_testing

**Comment by weilbith** [comment]
@agatsoh thanks for your test! :pray:

**Review by taleldayekh — APPROVED** [review]
Looks good code wise 👍🏼  Did not test it since @agatsoh already did. Thank you @weilbith !

**Comment by weilbith** [comment]
@taleldayekh what is your opinion about the question I explicitly asked you in the PR description? Maybe you should give it a fast try. At least just visiting the account root route.

**Comment by taleldayekh** [comment]
@weilbith Would be cool if you could adjust two things.
- Get rid of the scrollbar by making all content fit on the page when the account screen is not on mobile. Looks pretty bad with the scrollbar.

Screenshot 2021-02-16 at 17 39 27

- Make the `Main ETH` and `Raiden ETH` display on one line on mobile, you can see the screenshot in my PR #2563 
Screenshot 2021-02-16 at 17 39 47

**Comment by weilbith** [comment]
I targeted all the written feedback and also what we have discussed in the meeting (e.g. the version info should stick at the bottom and the transfer button should not be sticky). I'll now rebase this branch to integrate all fixup commits. Would be cool if @taleldayekh could then give me a final approval (might be optimistic).

PS: The scrollbar issue shown above was only in Chromium. This means that some sizes appear to differ on browsers. Eventually we need to investigate here to make it independent.

**Review by taleldayekh — APPROVED** [review]
Nice, the scrollbar is gone! Just one small thing, can you make the `Raiden ETH` display without line break when displayed on desktop? It does not matter if the balance numbers and address are not aligned but I think all text should be on one line.

Screenshot 2021-02-18 at 11 51 10

Other than that great, and thanks for making a proper fix.

**Comment by weilbith** [comment]
@taleldayekh are you sure that you tested the correct version? I saw this myself and just fixed it during the last rebase.

**Comment by taleldayekh** [comment]
> @taleldayekh are you sure that you tested the correct version? I saw this myself and just fixed it during the last rebase.

Let me try again.

**Comment by taleldayekh** [comment]
> @taleldayekh are you sure that you tested the correct version? I saw this myself and just fixed it during the last rebase.

Yes, you are right, I've must have checked an older version. Looks perfect, great job! ❤️ Go ahead and merge.
